_drafts/2020-07-27-data_science_plotting_notebook_template.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Jupyter Notebook Templates for Data Science: Plotting"
+description: >
+  Jupyter notebooks are great for data exploration; jumpstart your
+  visualizations with this plotting notebook!
+image: /files/jupyter-library/jupiter_red_spot_juno.jpg
+image_alt: >
+  A photo of the Library of Congress in 1902.
+categories: 
+  - jupyter
+  - my_projects
+---
+
+{% capture file_dir %}/files/jupyter-library/{% endcapture %}
+
+{% include lead_image.html %}
+
+I recently released my [Jupyter Notebook Template Library][library]. Its goal
```

Comment:
Not generally a fan of opening with "I" but I don't have an obvious re-word here